packages/lab/stories/form-field-next/form-field-next.doc.stories.mdx

```diff
@@ -0,0 +1,152 @@
+  CharacteristicUsage,
+} from "css-variable-docgen-components";
+import HelpAndSupport from "docs/blocks/help-and-support.doc.mdx";
+
+<Meta
+  title="Documentation/Lab/Form Field Next"
+  component={FormFieldNext}
+  parameters={{
+    viewMode: "docs",
+  }}
+/>
+
+# Form Field Next
+
+Form Field is a wrapper for UI controls that are typically found in a form (for example, Input, Combobox or Radio Button). It is required to make these controls accessible by providing them with a visible label, validation control and states and descriptive text elements.
+
+<Canvas>
+  <Story id="lab-form-field-next--default" />
+</Canvas>
+
+# Examples
+
+All examples demonstrate Salt form controls inside the Form Field, including Input, Checkbox and Radio Button. These components are available as part of the Salt component library in addition to Form Field.
+
+## Form Controls
+
+Form field can wrap any Salt UI control that is typically found in a form, such as Input, Checkbox and Radio Button components.
+
+<Canvas>
+  <Story id="lab-form-field-next--with-controls" />
+</Canvas>
+
+## Form Field Label
+
+Form Field provides a visible label which should be used to describe the form control in the field. The label is placed above the form control by default.
+
+Display the label via the `FormFieldLabel` component.
+
+<Canvas>
+  <Story id="lab-form-field-next--skeleton" />
+</Canvas>
+
+### Guidance
+
+- The label should be easy to scan and be no more than 3 words long.
+- Use the label to describe the information/data the field is collecting (e.g., Name, Location, Currency).
+- Use `FormFieldHelperText` to provide additional information and context to help the user complete the field. See the ‘Helper Text’ section below.
+
+## Left Form Field Label Placement
+
+The Form Field Label can be positioned to the left of the form control via the prop `labelPlacement`.
+
+<Canvas>
+  <Story id="lab-form-field-next--label-left" />
+</Canvas>
+
+### Guidance
+
+Place your label to the left of the control if your application is limited in vertical real estate or if you have a very data heavy, dense form, where it’s preferable to have as many fields visible as possible without scrolling.
+
+## Form Field Helper Text
+
+Form Field can display Helper Text using the `FormFieldHelperText` component.
+
+<Canvas>
+  <Story id="lab-form-field-next--helper-text" />
+</Canvas>
+
+### Guidance
+
+- Use Form Field Helper Text to provide additional guidance to users, such as an instruction (when it’s not obvious what action the user must take), or if you need to show an example of acceptable formats to increase the likelihood of successful entry.
+- Keep your Form Field Label short by moving guidance to the Helper Text.
+
+## Disabled
+
+Form Field can be disabled via the prop `disabled`.
+
+<Canvas>
+  <Story id="lab-form-field-next--disabled" />
+</Canvas>
+
+### Guidance
+
+- The prop should be applied to the field, not the control(s) it contains, for expected results.
```

Comment:
Should we remove the plural for controls here since our current guidance is to only use one?